presto-main/src/test/java/com/facebook/presto/execution/MockQueryExecution.java

```diff
@@ -75,7 +92,7 @@ public void complete()
     @Override
     public QueryId getQueryId()
     {
-        throw new UnsupportedOperationException();
+        return queryId.orElseThrow(UnsupportedOperationException::new);
```

Comment:
Too many callsites create MockQueryExecution without passing in a query id. I can default one if that's acceptable.